systemtest/src/main/java/io/strimzi/systemtest/Constants.java

```diff
@@ -409,8 +409,8 @@ public interface Constants {
     String LOAD_BALANCER_CLEANUP = "service.kubernetes.io/load-balancer-cleanup";
 
     // main namespace for Cluster Operator deployment
-    String INFRA_NAMESPACE = "infra-namespace";
-    String METRICS_SECOND_NAMESPACE = "second-metrics-cluster-test";
+    String CO_NAMESPACE = "co-namespace";
+    String TEST_SUITE_NAMESPACE = "test-suite-namespace";
```

Comment:
> In case all tests are @ParallelTest, or @IsolatedTest namespace test-suite-namespace is used ?

Yes.

> And apart from that if all tests are @ParallelNamespaceTest we should use co-namespace for cluster operator and something like for respective namespaces ?

You do not need to do it because it's already implemented [1]. Thus, if you use test storage in your test (now convention) there should be implicit `Constants.TEST_SUITE_NAMESPACE`. 

> More or less is this a rule of thumb as you are mentioning briefly that you are keeping co-namespace in some cases?

I keep `co-namespace` for upgrade and downgrade test cases and maybe some edge cases but majority of test cases should follow the same pattern from `TestStorage` (i.e., using `Constants.TEST_SUITE_NAMESPACE`)




[1] - https://github.com/strimzi/strimzi-kafka-operator/pull/8881/files#diff-70e45c4283fca95208daaace1fc0b12726b2d83759b56dd1d6b106a30e9eda71R55